cpp/src/transform/jit/kernel.cu

```diff
@@ -37,27 +38,45 @@ namespace transformation {
 namespace jit {
 
 template <typename T, int32_t Index>
-struct accessor {
+struct column_accessor {
   using type                     = T;
   static constexpr int32_t index = Index;
 
-  static __device__ decltype(auto) element(cudf::mutable_column_device_view_core const* views,
+  static __device__ decltype(auto) element(cudf::mutable_column_device_view_core const* outputs,
                                            cudf::size_type row)
   {
-    return views[index].element<T>(row);
+    return outputs[index].element<T>(row);
   }
 
-  static __device__ decltype(auto) element(cudf::column_device_view_core const* views,
+  static __device__ decltype(auto) element(cudf::column_device_view_core const* inputs,
                                            cudf::size_type row)
   {
-    return views[index].element<T>(row);
+    return inputs[index].element<T>(row);
   }
 
-  static __device__ void assign(cudf::mutable_column_device_view_core const* views,
+  static __device__ void assign(cudf::mutable_column_device_view_core const* outputs,
                                 cudf::size_type row,
                                 T value)
   {
-    views[index].assign<T>(row, value);
+    outputs[index].assign<T>(row, value);
+  }
+};
+
+template <typename T, int32_t Index>
```

Comment:
Why is our index type `int32_t` and not `cudf::size_type` or `std::size_t`?

Reply:
it's an index into function arguments, I'd say int32_t/uint32_t is the right type.
I was informed we preferred signed types over unsigned types.

Reply:
Thanks. That seems fine to me.